### DISCUSSION TITLE
You need to remember your vault main credential if you use device unlock recognition

### DISCUSSION THREAD

**Discussion by nollinvoyd** [Kunzisoft/KeePassDX#1772] [Q&A]
Samsung galaxy tab s5e

Everything was fine until yesterday.  I can login with the password, but when it comes to biometrics, I can't get beyond this

With "Device unlock link"

![keepass login message](https://github.com/Kunzisoft/KeePassDX/assets/20844322/1c3a8a2e-3da1-485e-9fb0-691856368c9d)

![20240228_115334](https://github.com/Kunzisoft/KeePassDX/assets/20844322/ef3a999a-4b89-41b6-acc1-07063f0c31f9)

"unavailable"?

![Screenshot_20240228-114057_KeePassDX](https://github.com/Kunzisoft/KeePassDX/assets/20844322/9014c0d1-55df-4ef8-b569-60f797bb8021)

I tried enabling this, then I couldn't login at all, even after I disabled it.  I had to uninstall and reinstall to get back in, but biometrics still not working.

![Screenshot_20240228-141227_KeePassDX](https://github.com/Kunzisoft/KeePassDX/assets/20844322/9d75f53c-0398-42f4-9487-34b7503e4686)

One other thing.  I can't edit this

![20240228_144853](https://github.com/Kunzisoft/KeePassDX/assets/20844322/3cf3aedf-9305-463f-9fe2-bb25411b7d06)

**Comment by J-Jamet** [comment]
https://github.com/Kunzisoft/KeePassDX/issues/1774
